### PR TITLE
Ability to enable/disable specific languages

### PR DESCRIPTION
**Pull request by michaelfarrell76**
## Related Issues

- links https://transcend.height.app/T-22

- New: airgap.js data attribute `data-languages` allows for specifying the specific set of languages that should be shown in the `LanguageOptions` view state. `data-languages="en,es-ES,fr-FR"` can be used to enable english, spanish and french options only. `data-languages="en"` can be used to enable english only. When only one option is enabled, the translation button is disabled entirely.

Screenshot 2023-08-10 at 3 36 27 PM
Screenshot 2023-08-10 at 3 37 32 PM

### CODE REVIEW

**Review by eligrey — DISMISSED** [review]
lgtm with a small nit: I think this should be `data-supported-languages` or `data-languages`

**Comment by michaelfarrell76** [comment]
@eligrey ill update to `data-languages`!

File: src/config.ts [hunk 1]
```diff
@@ -91,13 +100,22 @@ export function getMergedConfig(): ConsentManagerConfig {
   config.privacyPolicy ??= privacyPolicy;
   config.secondaryPolicy ??= secondaryPolicy;
   config.dismissedViewState ??= dismissedViewState;
+  config.languages ??= languages;
+
+  // Determine the language settings to use
+  const existingLanguages = config.languages ? config.languages.split(',') : [];
```
Comment (eligrey) [inline src/config.ts:106]: Do we want to allow optional whitespace?

```suggestion
  const existingLanguages = config.languages ? config.languages.split(/\s*,\s*/) : [];
```

**Review by eligrey — DISMISSED** [review]

File: src/config.ts (hunk 1 above)
Comment (michaelfarrell76) [reply src/config.ts]: added - i normally do .split(',').map((x) => x.trim()).filter((x) => !!x) but this is probably more efficient lol

**Review by eligrey — APPROVED** [review]

**Pull request merged**